catalog/core/catalog-core-camelcomponent/pom.xml

```diff
@@ -100,6 +100,10 @@
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.hazelcast</groupId>
+            <artifactId>hazelcast</artifactId>
```

Comment:
Should keep an eye out on CVE and OWASP vulnerabilities prior to merging, if we go EHCache as well